beacon-chain/orchestrator/client.go

```diff
@@ -0,0 +1,57 @@
+package orchestrator
+
+import (
+	"context"
+	"fmt"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+const confirmVanBlockHashesMethod = "orc_confirmVanBlockHashes"
+
+type Client interface {
```

Comment:
@mxmar Define another interface for method `ConfirmVanBlockHashes(ctx context.Context, request []*vanTypes.ConfirmationReqData) (response []*vanTypes.ConfirmationResData, err error)` and embed the interface into this Client interface.

Reply:
@atif-konasl In other words you would like to separate RPC logic and RPC requests like `rpc please confirm these blocks` right?

Reply:
yes right. `blockchain` package will not invoke these `Dial`, `DialContext` etc methods. So you just need an interface which only have this `ConfirmVanBlockHashes(ctx context.Context, request []*vanTypes.ConfirmationReqData) (response []*vanTypes.ConfirmationResData, err error)` method definition so that `blockchain` package can call this method.

Reply:
got it, done.